test/e2e-tests.sh

```diff
@@ -189,7 +205,12 @@ kustomize build overlays/dev-locked-down --load_restrictor=LoadRestrictionsNone
 kustomize build overlays/dev-openshift --load_restrictor=LoadRestrictionsNone || fail_test "Failed to run kustomize on overlays/dev-openshift"
 kustomize build overlays/dev-openshift-locked-down --load_restrictor=LoadRestrictionsNone || fail_test "Failed to run kustomize on overlays/dev-openshift-locked-down"
 
-test_overlay dev "proxy"
-test_overlay dev-locked-down "kubectl"
+# test overlays
+test_dashboard overlay proxy dev
+test_dashboard overlay kubectl dev-locked-down
+
+# test installer
```

Comment:
Is the plan to remove the `overlay` tests eventually since the installer is doing the same thing under the covers?

Reply:
probably later if we promote the installer and deprecate the overlays. as long as we release manifests based on overlays we should keep the tests